app/controllers/notification_recipients_controller.rb

```diff
@@ -7,11 +7,12 @@ class NotificationRecipientsController < Api::V2::BaseController
   def index
     @notifications = NotificationRecipient.
       where(:user_id => User.current.id, :notification_id => Notification.active).
-      order(created_at: :desc).
+      order(:created_at).
       eager_load(:notification, :notification_blueprint)
 
     render :json => {
-      :notifications => @notifications.paginate(paginate_options).map(&:payload)
+      :notifications => @notifications.paginate(paginate_options).map(&:payload),
+      :total => @notifications.count
```

Comment:
@ohadlevy said to return the total, since it's breaking the tests, so i reverted the entire file.